From 766c6544a6cc6fd7ee9a5e164754350735c22506 Mon Sep 17 00:00:00 2001
From: WofWca <wofwca@protonmail.com>
Date: Sun, 27 Sep 2026 12:16:27 +0400
Subject: [PATCH] improvement: increase default random velocity CVAR

It might have been a mistake to make it so small.
We still need a bit of that "explosion" factor.
Bare linear speed is not enough.
---
 code/cgame/cg_cvar.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/code/cgame/cg_cvar.h b/code/cgame/cg_cvar.h
index 6475ec297..3a734a6b5 100644
--- a/code/cgame/cg_cvar.h
+++ b/code/cgame/cg_cvar.h
@@ -105,7 +105,7 @@ CG_CVAR( cg_gibsExplosionDistanceOffset, "cg_gibsExplosionDistanceOffset", "-3.0
 	"For comparison, player height and width are 56 and 30 units.\n"
 	"This lets you increase the spread. "
 	"This doesn't affect the damage and knockback.\n" )
-CG_CVAR( cg_gibsRandomVelocityFromKnockback, "cg_gibsRandomVelocityFromKnockback", "0.3", CVAR_ARCHIVE,
+CG_CVAR( cg_gibsRandomVelocityFromKnockback, "cg_gibsRandomVelocityFromKnockback", "0.5", CVAR_ARCHIVE,
 	"Scale how much random velocity (spread) the gibs gain "
 	"from damage (knockback)." )
 CG_CVAR( cg_gibsVerticalVelocityFromKnockback, "cg_gibsVerticalVelocityFromKnockback", "0.2", CVAR_ARCHIVE,
-- 
2.56.0

